metaflow/main_cli.py

```diff
@@ -349,83 +348,79 @@ def sandbox(profile):
     # Persist to a file.
     persist_env(env_dict, profile)
 
-@configure.command(help='Get Metaflow up and running on your own AWS environment.')
+@configure.command(help='Configure Metaflow to access self-managed AWS resources.')
 @click.option('--profile', '-p', default='',
-              help="Optional user profile to allow storing multiple "
-                   "configurations. Please `export METAFLOW_PROFILE` to "
-                   "switch between profile configuration(s).")
+              help='Configure a named profile. Activate the profile by setting ' 
+                    '`METAFLOW_PROFILE` environment variable.')
 def aws(profile):
+    def cyan(string):
+        return click.style(string, fg='cyan')
+
+    def yellow(string):
+        return click.style(string, fg='yellow')
+
     prompt_config_overwrite(profile)
-    if click.confirm('Have you setup your ' +\
-                     click.style('AWS credentials?', fg='cyan')):
+    if not click.confirm('Already configured ' + cyan('AWS access credentials') + '?',
+          default=True):
+        echo('\nSetup your access credentials by following this guide: ', nl=False)
+        echo('https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html',
+             fg='cyan')
+    else:
         env_dict = {}
         # Datastore configuration.
-        use_s3 = click.confirm('\nDo you want to use AWS S3 as your datastore?',
+        use_s3 = click.confirm('\nConfigure Amazon '
+                                'S3 as default datastore (required for AWS Batch)?',
                               default=True, abort=False)
         if use_s3:
-            echo('\tAWS S3', fg='cyan')
-            datastore_s3_root =\
-                click.prompt('\tPlease enter the bucket prefix to use for your '
-                             'flows')
-
-            datatools_s3_root =\
-                click.prompt('\tPlease enter the bucket prefix to use for your '
-                             'data',
-                             default='%s/data' % datastore_s3_root)
-
             env_dict['METAFLOW_DEFAULT_DATASTORE'] = 's3'
-            env_dict['METAFLOW_DATASTORE_SYSROOT_S3'] = datastore_s3_root
-            env_dict['METAFLOW_DATATOOLS_SYSROOT_S3'] = datatools_s3_root
-
-            # Batch configuration (only if S3 is being used).
+            env_dict['METAFLOW_DATASTORE_SYSROOT_S3'] =\
+                click.prompt('\t' + cyan('METAFLOW_DATASTORE_SYSROOT_S3:') + 
+                  ' Amazon S3 url for metaflow datastore (s3://<bucket>/<prefix>)')
+
+            env_dict['METAFLOW_DATATOOLS_SYSROOT_S3'] =\
+                click.prompt('\t' + cyan('METAFLOW_DATATOOLS_SYSROOT_S3:') + 
+                  yellow(' (optional)') + ' Amazon S3 url for metaflow datatools (s3://<bucket>/<prefix>)',
```

Comment:
Nit: Do you need this again, since you suggest a default in brackets right after this. I vote for removing it, since it seems pretty clear with the actual default value.
```
METAFLOW_DATATOOLS_SYSROOT_S3: (optional) Amazon S3 url for metaflow datatools (s3://<bucket>/<prefix>) [s3://foo/metaflow/data]
```

Reply:
This is to follow a consistent pattern throughout.